compiler/control/OMROptions.hpp

```diff
@@ -234,7 +234,6 @@ enum TR_CompilationOptions
    TR_EnableYieldVMAccess                 = 0x02000000 + 4,
    TR_DisableNoVMAccess                   = 0x04000000 + 4,
    TR_DisableStoreSinking                 = 0x08000000 + 4,
-   TR_EnableJVMPILineNumbers              = 0x10000000 + 4,
```

Comment:
Please don't delete this outright.  Instead, replace it with an "Available" comment to encourage re-use, like in:
https://github.com/eclipse/omr/blob/b406ab2841765a3a182850d8884a9c0297845e00/compiler/control/OMROptions.hpp#L104

Reply:
Fixed